pkg/deps/task.go

```diff
@@ -0,0 +1,30 @@
+package deps
+
+import (
+	"context"
+
+	"github.com/skygeario/skygear-server/pkg/auth/config"
+	"github.com/skygeario/skygear-server/pkg/core/intl"
+	"github.com/skygeario/skygear-server/pkg/task"
+	"github.com/skygeario/skygear-server/pkg/task/executors"
+	"github.com/skygeario/skygear-server/pkg/task/queue"
+)
+
+func ProvideCaptureTaskContext(ctx context.Context, config *config.Config) queue.CaptureTaskContext {
+	return func() *task.Context {
+		return &task.Context{
+			Config:                config,
+			PreferredLanguageTags: intl.GetPreferredLanguageTags(ctx),
```

Comment:
Actually the existing tasks have been refactored to not depend on Config or PreferredLanguageTags. We can remove them.